client/my-sites/plugins/use-plugins/index.ts

```diff
@@ -78,7 +78,7 @@ const usePlugins = ( {
 			: useWPORGInfinitePlugins;
 
 	const wporgPluginsOptions = {
-		locale,
+		locale: locale || 'en',
```

Comment:
Need to revisit this as it's probably not needed anymore.

Reply:
Agree, needing to fallback to english here is most likely masking a [bigger issue](https://github.com/Automattic/wp-calypso/pull/66087/files?w=1#r934126347). :smile: